### DISCUSSION TITLE
ShinyFramework question

### DISCUSSION THREAD

**Discussion by mjphillips1981** [shinyorg/shiny#850] [Q&A]
Hello,   So this question pertains to ShinyFramework but since there is no discussion tab over there, I thought I would put this here.  

I have put together an app using ShinyFramework nuget. All seems to be working well until I try and build a release version for iOS.  I get this error below.   All works fine when I run a debug session in iOS and debug/release on Android.   Any help would be appreciated. 

Can't resolve the reference 'ReactiveUI.ReactiveCommand`2<System.Reactive.Unit,System.Reactive.Unit> ReactiveUI.ReactiveCommand::CreateFromTask(System.Func`1<System.Threading.Tasks.Task>,System.IObservable`1<System.Boolean>,System.Reactive.Concurrency.IScheduler,System.Reactive.Concurrency.IScheduler)', referenced from the method 'System.Windows.Input.ICommand Shiny.BaseViewModel::LoadingCommand(System.Func`1<System.Threading.Tasks.Task>,System.String,System.Boolean)' in 'ReactiveUI, Version=16.3.0.0, Culture=neutral, PublicKeyToken=null'.

**Comment by aritchie** [comment]
Shiny Framework isn't something I'm officially supporting at this time.  

In this case, your issue appears to be the linker removing part of reactiveui.  You can try installing the reactiveui nuget yourself, otherwise, you will have to read the documentation from microsoft on how to fix linker issues like this.

**Reply by mjphillips1981** [reply]
Thank you for pointing me in a direction...
First thing I tried was the Reactive Nuget (actually just before you responded).  Did not work.
Then I was reading about the linker.  Switching to Don't Link made everything work.  I will continue down this path.  

Thank you Allan!

**Reply by mjphillips1981** [reply]
Switch my commands over to prism delegate commands and the issue went away.  Definitely something going on with the linker.
